contents/blog/best-adobe-analytics-alternatives.mdx

```diff
@@ -0,0 +1,458 @@
+  <ComparisonRow column1={true} column2={true} feature="Cohort analysis" description="Combine users into cohorts for analysis" />
+  <ComparisonRow column1={true} column2={false} feature="Session replay" description="Watch real users on your site; discover friction points" />
+  <ComparisonRow column1={true} column2={false} feature="Heatmaps" description="Visualize where users click and interact on your site" />
+  <ComparisonRow column1={false} column2={false} feature="A/B testing" description="Run multivariate tests and see the impact of changes with custom goals and reports" />
+  <ComparisonRow column1={true} column2={false} feature="Script install" description="Start capturing data by adding a simple script to your site" />
+  <ComparisonRow column1={false} column2={false} feature="EU hosting" description="Store your data in the EU for compliance" />
+  <ComparisonRow column1={false} column2={false} feature="Open source" description="Audit code, contribute to roadmap, and build integrations" />
+</ComparisonTable>
+
+### Why do companies use Heap?
+
+According to G2 reviews, companies enjoy these three areas of Heap:
+
+1. **Autocapture:** Non-technical users love how easy autocapture makes tracking on their site. Along with the element data included, this provides a huge amount of useful analytics data with little setup.
+
+2. **Simple setup:** It does not take a big technical effort to set Heap up. Users can add a single script and begin collecting data. They then make it easy to visualize that data through user paths, funnels, and session replays.
+
+3. **Streamlining analysis:** By having analytics and session replay data in one place, Heap makes it easy to understand the usage of their app or site. This replaces interviews or user testing and makes the development cycle faster.
+
+> #### Bottom line
+> Because of its similar focus on product analytics along with autocapture and being self-serve, Heap makes for a good Adobe Analytics alternative. It still lacks the simple web analytics view that Adobe Analytics users might want.
+
+<br />
+
+## 5. Google Analytics
+
+- **Founded:** 2005
+- **Similar to:** Matomo
+- **Typical users:** Marketing and business teams reliant on Google's ecosystem
+- **Typical customers:** Small businesses and massive enterprises
+
+![Google Analytics](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_10_02_at_14_54_37_25bd7082b7.png)
+
+### What is Google Analytics?
+
+[Google Analytics](/blog/posthog-vs-ga4) has long been the go-to choice for website and app analytics thanks to Google's huge market share, a large amount of informational content, and its connection with the rest of the Google suite.
+
+Google Analytics recently switched fully over from session-based Universal Analytics (GA3) to event-based GA4. This also introduced conversion funnels and retention tables that product teams are fans of. 
```

Comment:
If you felt like dunking on GA at this juncture, I have firsthand testimony that the transition made a lot of teams MISERABLE, and some of them abandoned their analytics altogether instead of figure out what went wrong.

But I defer to your sense of propriety!

Reply:
I'll adjust to say that some don't like it either 😄